pubtools/_quay/signer_wrapper.py

```diff
@@ -52,7 +52,7 @@ def __init__(
             settings (dict): Settings for the signer.
         """
         self.config_file = config_file
-        self.settings = settings
+        self.settings = settings or {}
```

Comment:
Can we replace this with `settings: Dict[str, Any] = {}` in method signature?

Reply:
yeah, that would be ok from typing perspective but having empty dict or list as default in kwargs is considered as dangerous according to some PEP